### DISCUSSION TITLE
Nemo action "Send To"?

### DISCUSSION THREAD

**Discussion by axel-rank** [linuxmint/discussions#247] [Ideas]
It would be nice and practical to add the "Send To" Feature to Nemo Context Menu.
The Feature is available in nearly all other File Managers (Caja, Nautilus etc.).

**Comment by rcalixte** [comment]
Most of the similar functionality is [here](https://cinnamon-spices.linuxmint.com/actions/view/2). It is named differently and there are some subtle differences. Is there anything you've found that isn't working for you? Are there additional features that you think are missing?
